CheckDrive.Api/CheckDrive.Application/Services/Authorization/AuthService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CheckDrive.Application.DTOs.Identity;
+using CheckDrive.Application.Interfaces.Authorization;
+using CheckDrive.Domain.Exceptions;
+using Microsoft.AspNetCore.Identity;
+
+namespace CheckDrive.Application.Services.Authorization;
+
+public sealed class AuthService : IAuthService
+{
+    private readonly IJwtTokenGenerator _jwtTokenGenerator;
+    private readonly UserManager<IdentityUser> _userManager;
+
+    public AuthService(
+        IJwtTokenGenerator jwtTokenGenerator,
+        UserManager<IdentityUser> userManager)
+    {
+        _jwtTokenGenerator = jwtTokenGenerator ?? throw new ArgumentNullException(nameof(jwtTokenGenerator));
+        _userManager = userManager ?? throw new ArgumentNullException(nameof(userManager));
+    }
+
+    public async Task<string> LoginAsync(LoginDto loginDto)
```

Comment:
Validate methods input for null

Reply:
> Validate methods input for null

I have a question , then I validate if this input is null then I throw 
ArgumentNullException or InvalidLoginException